mobile/lib/modules/shared_link/providers/shared_link.provider.dart

```diff
@@ -0,0 +1,89 @@
+import 'package:hooks_riverpod/hooks_riverpod.dart';
+import 'package:immich_mobile/modules/shared_link/services/shared_link.service.dart';
+import 'package:openapi/api.dart';
+
+class SharedLinksNotifier
+    extends StateNotifier<AsyncValue<List<SharedLinkResponseDto>>> {
+  final SharedLinkService _sharedLinkService;
+
+  SharedLinksNotifier(this._sharedLinkService) : super(const AsyncLoading()) {
+    fetchLinks();
+  }
+
+  fetchLinks() async {
```

Comment:
could you add return types to all these functions in the file? should be `Future<void>`

Reply:
Updated.